tachyon/math/finite_fields/prime_field_generic.h

```diff
@@ -277,7 +277,7 @@ class PrimeField<_Config, std::enable_if_t<!_Config::kIsSpecialPrime>> final
   }
 
   constexpr PrimeField& SlowMulInPlace(const PrimeField& other) {
-    BigInt<N * 2> r = value_.Mul(other.value_);
+    BigInt<N * 2> r = value_.MulExtend(other.value_);
```

Comment:
To standardize <2 * N>
```suggestion
    BigInt<2 * N> r = value_.MulExtend(other.value_);

Reply:
Does line 280 in the file `tachyon/math/finite_fields/prime_field.h` also need to be updated?

Reply:
That was removed in https://github.com/kroma-network/tachyon/pull/363